pkg/kubehound/models/store/role.go

```diff
@@ -12,4 +12,5 @@ type Role struct {
 	Namespace    string              `bson:"namespace"`
 	Rules        []rbacv1.PolicyRule `bson:"rules"`
 	Ownership    OwnershipInfo       `bson:"ownership"`
+	K8           rbacv1.Role         `bson:"k8"`
```

Comment:
do we need this?

Reply:
I had something in mind, but I think I no longer using it. Removing it.